src/lib/containers/HeaderCard.tsx

```diff
@@ -29,6 +29,15 @@ const HeaderCard: React.FunctionComponent<HeaderCardProps> = ({
   secondaryLabelLimit,
   isAlignToLeftNav
 }) => {
+  // update page title and description based on header card values
+  if (title && document.title !== title) {
+    document.title = title
+  }
+  const descriptionElement:Element|null = document.querySelector('meta[name="description"]')
```

Comment:
Are we covering all cases of setting the document title in the portals? My concern is that a page will have the wrong document title and metadata with this sort of modification existing on a per component basis, as opposed to in the portal code.